### ISSUE TITLE
The 'note' class has no special formatting

### ISSUE THREAD

**Issue by ndjc** [oasis-tcs/tab-respec#49]
With ReSpec 2.1.22, the 'note' class has no special formatting to distinguish it from other text. Since notes are informative and not normative, some distinguishing formatting seems appropriate.

There also appears to be no special spacing added before a note header, nor any different formatting of the note title.

We should ascertain OASIS best practices for notes in specs, and set up an appropriate CSS style.

**Comment by berezovskyi** [comment]
Let's look at this when we have some PNs ready for publication. I will coordinate with David as the PN editor and Paul as necessary.

**Comment by berezovskyi** [comment]
Proposed formatting:

```
div.note {
    margin-left: 3em;
    border-left: solid #900 4px;
    padding: 3px 0px 0px 10px;
}

.note > .note-title {
    font-weight: 700;
}
```

**Comment by ndjc** [comment]
The problem was not that there was no css for examples and notes, but that the ReSpec code to add that css was not working. I've fixed that in ReSpec 2.1.26, but @berezovskyi you should now review the css elements `tab-respec/js/css/examples.css` and `tab-respec/js/css/issues-notes.css`.

To avoid making too many changes at the same time, I have **not** addressed the same potential issue in loading the styles `bp.css`, `conformance.css`, `highlight.css`, or `webidl-oldschool.css`.
